ckcp/tekton-results/openshift/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
All yaml must start with `---`, otherwise yamllint will complain in the CI. Occurs multiple times in the PR.

Reply:
Updated yaml